api/conf/settings/oauth2.py

```diff
@@ -3,13 +3,12 @@
 from conf.settings.base import INSTALLED_APPS, MIDDLEWARE
 
 
-INSTALLED_APPS += [
-    "oauth2_provider",
-]
+if "oauth2_provider" not in INSTALLED_APPS:
+    INSTALLED_APPS.append("oauth2_provider")
 
-MIDDLEWARE += [
-    "oauth2_provider.middleware.OAuth2TokenMiddleware",
-]
+middleware = "oauth2_provider.middleware.OAuth2TokenMiddleware"
+if middleware not in MIDDLEWARE:
+    MIDDLEWARE.append(middleware)
```

Comment:
Would it make sense to make `INSTALLED_APPS` and `MIDDLEWARE` sets so we could just call `.add`, or is order important?

Reply:
The order is important. For example, the CORS middleware specifies that it should be set  as early in the list as possible, hence https://github.com/WordPress/openverse/pull/2075/files#diff-1e4c90df768f3514290795d5e58d3b561644bd37e4dfdb9938b60083dd062d30R45